examples/distributed_simulation/main.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: ASSUME Developers
```

Comment:
the whole idea is really great, but I doubt anyone would use it as it is very unclear what exactly is happening. Please always keep in mind, that we are cretaing this not for our own use, so only us can understand what is written here. Especially since this part is in the examples, here should be clear comments and details what is happening and why